tests/jest/unit/modules/atomic-widgets/hooks/data/duplicated-styles/utils/set-global-containers.js

```diff
@@ -0,0 +1,15 @@
+export function setGlobalContainers( containers = [] ) {
```

Comment:
Do we need another file?

Reply:
If it is a general util, maybe it should be outside of duplicated-styles dir and please re-use it when ever need